app/cdash/tests/CMakeLists.txt

```diff
@@ -602,5 +602,12 @@ set_tests_properties(misassignedconfigure PROPERTIES DEPENDS /Feature/SubProject
 add_laravel_test(/Feature/RemoteWorkers)
 set_tests_properties(/Feature/RemoteWorkers PROPERTIES DEPENDS install_3)
 
+add_laravel_test(/Unit/app/Console/Command/ValidateXmlCommandTest)
+
+add_php_test(submissionvalidation)
+# Set this to RUN_SERIAL as we are changing the config and don't want to clobber other tests
+set_tests_properties(submissionvalidation PROPERTIES RUN_SERIAL TRUE)
+set_tests_properties(submissionvalidation PROPERTIES DEPENDS /Feature/AutoRemoveBuildsCommand)
```

Comment:
I've been trying to put all of the "independent" tests between `install_2` and `install_3`.  Want to move this somewhere around line 200 instead?